src/pages/Faq/FaqModal.js

```diff
@@ -0,0 +1,336 @@
+            </InputWrap>
+            <InputWrap>
+              <FormTitle>문의내용</FormTitle>
+              <Textarea
+                placeholder="문의내용을 입력해주세요"
+                onChange={handleForm}
+                name="content"
+                value={inputValue.content}
+              />
+            </InputWrap>
+            <ContactContainer>
+              <InputWrap>
+                <FormTitle>이메일</FormTitle>
+                <Input
+                  placeholder="문의내용을 입력해주세요"
+                  onChange={handleForm}
+                  type="email"
+                  name="email"
+                  value={inputValue.email}
+                />
+              </InputWrap>
+              <InputWrap>
+                <FormTitle>휴대폰 번호</FormTitle>
+
+                <Input
+                  placeholder="문의내용을 입력해주세요"
+                  type="text"
+                  onChange={handlePhone}
+                  name="phoneNumber"
+                  value={inputValue.phoneNumber}
+                  maxlength="8"
+                />
+              </InputWrap>
+            </ContactContainer>
+            <InputWrap>
+              <FormTitle>첨부파일</FormTitle>
+              <p>pdf 파일만 가능합니다.</p>
+              <FileLabel htmlFor="pdfFile">파일 추가하기</FileLabel>
+              <FileUpload
+                type="file"
+                id="pdfFile"
+                placeholder="파일추가"
+                accept=".pdf"
+                onChange={handleFile}
+              />
+              <UploadList>
+                {files.map((file, idx) => {
+                  return (
+                    <UploadFile key={idx}>
+                      <UploadFileName>{file.name}</UploadFileName>
+                      <UploadFileDelete onClick={() => removeFile(idx)} />
+                    </UploadFile>
+                  );
+                })}
+              </UploadList>
+            </InputWrap>
+            <AgreeContainer>
+              <AgreeBox
+                id="isAgree"
+                type="checkbox"
+                onChange={handleChecked}
+                value={inputValue.check}
+              />
+              <AgreeCheck htmlFor="isAgree">
+                개인정보 수집 및 이용동의
+              </AgreeCheck>
+
+              <AgreeContent>
+                주)코딩에 세상을 구한다(이하 ‘코세구’)은 아래와 같이 개인정보를
+                수집 및 이용합니다. <br />
+                1. 수집 · 이용 항목 문의 등록 시 다음정보를 수집 및 이용합니다.
+                - 이메일주소, 휴대전화번호 <br />
+                2. 수집 · 이용 목적 수집한 개인정보는 아래 목적으로 이용됩니다.
+                - 문의 확인 및 처리, 처리결과 전달, 문의 처리 시 필요에 따른
+                추가 안내 <br />
+                3. 보유 및 이용기간 지원자 계정 삭제 시 즉시 파기합니다.
+                (로그인하지 않고 접수한 문의는 문의처리 완료 후 6개월 뒤 자동
+                파기합니다.) <br />
+                개인정보 수집 이용에 동의하지 않으실 수 있으며, 동의하지 않는
+                경우 문의하기가 제한됩니다.
+              </AgreeContent>
+            </AgreeContainer>
+          </Form>
+          <BtnWrap>
+            <SubmitBtn onClick={handleSubmit} isValidaiton={validate}>
```

Comment:
```suggestion
            <SubmitBtn onClick={handleSubmit} disabled={validate}>
```
SubmitBtn 컴포넌트는 button의 역할을 하고 있는 컴포넌트기 때문에
disabled라는 button의 속성을 그대로 사용할 수 있습니다!